components/footer.jsx

```diff
@@ -76,47 +73,28 @@ const Footer = () => {
               <LinkRenderer key={link.path} link={link} />
             ))}
           </Col>
-          <Col xs={4}>
-            <strong>Documents</strong>
-            {documents.map((link) => (
-              <Link
-                key={link.path}
-                href={link.path}
-                className={`nav-link ${css(styles.footerlink)}`}
-                target="__blank"
-              >
-                {link.name}
-              </Link>
-            ))}
-          </Col>
         </Row>
       </Container>
-      <div className={css(styles.divider)} />
+      <div className={styles.divider} />
     </Container>
   );
 };
 
-const styles = StyleSheet.create({
-  footerlink: {
-    padding: "0.1rem 0 !important",
-  },
-  divider: {
-    display: "inline-block",
-    height: "0.05rem !important",
-    width: "85vw !important",
-    backgroundColor: "#bdbdbd !important",
-    margin: "1.75rem 0 !important",
-  },
-  communitybar: {
-    maxWidth: "575.98px",
-  },
-  communitylink: {
-    filter: "saturate(0) invert(1)",
-    display: "inline-block",
-    ":hover": {
-      filter: "invert(0.65)",
-    },
-  },
-});
-
 export default Footer;
+
+/*
```

Comment:
We don't need these comments worse case scenario you pin or open an new github issue with 
a due date on the action.